contrib/ci/dependencies.xml

```diff
@@ -632,6 +632,11 @@
       <package variant="x86_64" />
     </distro>
   </dependency>
+  <dependency id="gpgme-devel">
+    <distro id="fedora">
+      <package variant="x86_64">gpgme-devel</package>
+    </distro>
+  </dependency>
```

Comment:
can we just wait a few days for 2.0.1 to hit the Fedora remotes?

Reply:
Yeah. I'll drop that commit. I mostly rebased to see if anything else recent explodes.